presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -1158,8 +1198,21 @@ public Object prune(Object value)
         }
     }
 
-    private static List<List<?>> pruneValues(List<Type> types, List<List<?>> values, Map<Integer, List<Subfield>> requiredSubfields)
+    private static List<List<?>> pruneValues(List<Type> types, List<List<?>> values, Map<Integer, Map<Subfield, TupleDomainFilter>> columnFilters, Map<Integer, List<Subfield>> requiredSubfields, List<FilterFunction> filterFunctions)
     {
+        for (FilterFunction function : filterFunctions) {
```

Comment:
@wenleix, any suggestion on simulating the no-input, tuple domain, other filter order abstraction?